restructuredtext_lint/test/test.py

```diff
@@ -93,7 +95,33 @@ def test_invalid_link(self):
 
         This is a regression test for https://github.com/twolfson/restructuredtext-lint/issues/12
         """
-        filepath = __dir__ + '/test_files/invalid_link.rst'
+        filepath = os.path.join(__dir__, 'test_files', 'invalid_link.rst')
         errors = restructuredtext_lint.lint_file(filepath)
         self.assertIn('Anonymous hyperlink mismatch: 1 references but 0 targets.', errors[0].message)
         self.assertIn('Hyperlink target "hello" is not referenced.', errors[1].message)
+
+
+class TestRestructuredtextLintCLI(TestCase):
+    """ Tests for 'rst-lint' CLI comand """
+
+    def test_rst_lint_filepaths_not_given(self):
+        """The `rst-lint` command is available and prints error if no filepath was given."""
+        with self.assertRaises(CalledProcessError):
+            output = check_output('rst-lint')
+            self.assertIn('too few arguments', output)
+
+    def test_rst_lint_correct_file(self):
+        """The `rst-lint` command prints out 'X is clean' if rst file is correct."""
+        valid_file = os.path.join(__dir__, 'test_files', 'valid.rst')
+        output = str(check_output(('rst-lint', valid_file))).splitlines()
+        self.assertIn('{filepath} is clean'.format(filepath=valid_file), output[0])
+        self.assertEqual(len(output), 1)
+
+    def test_rst_lint_many_files(self):
+        """The `rst-lint` command accepts many rst file paths and prints respective information for each of them."""
+        with self.assertRaises(CalledProcessError) as e:
+            check_output('rst-lint ' + os.path.join(__dir__, 'test_files', '*rst'), shell=True)
```

Comment:
Please don't use concatenation. If someone has a space in their pathname, then the test will fail. Instead, pass these as separate arguments:

```python
check_output(('rst-lint ', os.path.join(__dir__, 'test_files', '*rst')), shell=True)
```